### PR TITLE
add a unix_timestamp extern

### PR DESCRIPTION
**Pull request by ZephyrTFA**
this is for use in https://github.com/tgstation/tgstation/pull/70830

### CODE REVIEW

File: src/time.rs [hunk 1]
```diff
@@ -35,3 +35,7 @@ byond_fn!(fn time_reset(instant_id) {
         Some("")
     })
 });
+
+byond_fn!(fn unix_timestamp() {
+    Some(std::time::SystemTime::now().duration_since(std::time::UNIX_EPOCH).unwrap().as_secs().to_string())
```
Comment (Mothblocks) [inline src/time.rs:40]:
```suggestion
    Some(std::time::SystemTime::now().duration_since(std::time::UNIX_EPOCH).unwrap().as_secs_f64().to_string())
```

We would want as much precision as possible for logs

**Review by ZeWaka — APPROVED** [review]
will [need](https://doc.rust-lang.org/cargo/reference/semver.html#minor-adding-new-public-items) a minor version bump

**Comment by Mothblocks** [comment]
We're gonna have a few PRs in the next batch so I'll make sure

**Comment by Mothblocks** [comment]
@ZephyrTFA You need to run cargo fmt

**Comment by ZephyrTFA** [comment]
> @ZephyrTFA You need to run cargo fmt

Not currently able to as I am at work, can I convince you to do it for me?

**Comment by Mothblocks** [comment]
I can't I'm at work too

**Comment by ZephyrTFA** [comment]
@Mothblocks formatted

**Pull request merged**